main.go

```diff
@@ -194,6 +203,27 @@ func cleanValueExpr(tokens hclwrite.Tokens) hclwrite.Tokens {
 	return trimNewlines(inside)
 }
 
+func cleanProviderExpr(tokens hclwrite.Tokens) hclwrite.Tokens {
+	if len(tokens) != 3 {
+		// We're only interested in plain quoted strings, which consist
+		// of the open and close quotes and a literal string token.
+		return tokens
+	}
+	oQuote := tokens[0]
+	strTok := tokens[1]
+	cQuote := tokens[2]
+	if oQuote.Type != hclsyntax.TokenOQuote || strTok.Type != hclsyntax.TokenQuotedLit || cQuote.Type != hclsyntax.TokenCQuote {
+		// Not a quoted string sequence, then.
+		return tokens
+	}
+	return hclwrite.Tokens{
+		{
+			Type:  hclsyntax.TokenIdent,
+			Bytes: []byte(strTok.Bytes),
+		},
```

Comment:
Due to how `body.SetAttributeRaw` works (it just trusts whatever it's given) this doesn't matter too much, but technically a `provider.alias` expression is three tokens: one ident, then a dot, and then another ident.

With that said, this tool's purpose is very specialized and so a shortcut like this shouldn't hurt, and I don't think it's worth the complexity of trying to parse the string to split it into the three separate tokens it would normally parse to.

Would you mind adding a comment here just to help any future person referring to this code see that this is not a good general example of using `hclwrite` and that we're only doing it as a shortcut for this specific case?

```go
// HACK: Technically a provider.alias sequence ought to be three separate
// tokens, because the dot is an operator, but only the `Bytes` part of this
// is relevant to our output anyway so we'll cheat and thus avoid the need
// to parse `strTok.Bytes`.

```